concrete/src/Entity/File/Version.php

```diff
@@ -1869,9 +1869,9 @@ public function getDetailThumbnailImage()
                 /** @var ThumbnailPlaceholderService $thumbnailPlaceholderService */
                 $thumbnailPlaceholderService = $app->make(ThumbnailPlaceholderService::class);
                 $result = $thumbnailPlaceholderService->getThumbnailPlaceholder($this, $type->getBaseVersion());
-            } else {
```

Comment:
if:
- `$this->getTypeObject()->supportsThumbnails()` is true
- `$this->fvHasDetailThumbnail === ThumbnailerInterface::THUMBNAIL_EXISTS` is false
- `$this->fvHasDetailThumbnail === ThumbnailerInterface::THUMBNAIL_GENERATING` is false

then `$result` won't be defined, right?